src/components/RecipeGallery.js

```diff
@@ -0,0 +1,74 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+
+
+function RecipeGallery({userInput}) {
+
+    const [recipeResults , setRecipeResults ] = useState([]);
+
+    const [recipeSearch, setRecipeSearch] = useState(null);
+
+    if(recipeSearch !== null) {
+        setRecipeSearch(userInput)
+    }
+
+    useEffect(() => {
```

Comment:
I noticed when I search a few things, it breaks with "cross-origin request blocked - CORS header 'Access-Control-Allow-Origin'" and "Uncaught (in promise)" errors. I believe one solution to resolving CORs is via using a proxy website, and you might need to add .catch() to handle failed API calls!